include/details.md

```diff
@@ -42,18 +43,22 @@ __Documentation__
     - {{ feature }}
     {% endfor %}
 
-{% if (features.programming_languages or features.environments) %}
-=== "Kernels"
+{% if (features.programming_languages or features.environments) %} 
+=== "Kernels :material-information-outline:{ title="Links to Kernels" }" 
     {% if features.programming_languages %}
     Programming Languages:
     {% for kernel in features.programming_languages %}
+    {% if Links[kernel] %}
```

Comment:
This will only work if the key in kernels_and_extensions.yaml matches exactly the word, people put in the data.yaml file. What about handling different capitalization? Versions at the end? You have to handle that here.

Reply:
Will remove the versions and as stated previously would " | lower" to change user input to lower case and have the keys lowered as well.

Reply:
Can not see any code that removes the version... do I miss something?